drizzlepac/haputils/make_poller_files.py

```diff
@@ -117,35 +133,39 @@ def locate_fitsfile(search_string):
          full file path + image name of specified search_string.
     """
     if search_string.endswith("_flt.fits") or search_string.endswith("_flc.fits"):  # Process search_string as a full filename
-        # Look for files in CWD first
-        if os.path.exists(search_string):
+        # Look in user-provided path (assuming they provided one)
+        if os.path.exists(search_string) and "/" in search_string:
```

Comment:
The use of the hard-coded value '/' here should be replaced instead with 'os.sep' in order to make this code more platform-independent.

Reply:
issue resolved by commit [6626a45](https://github.com/spacetelescope/drizzlepac/pull/1138/commits/6626a452591a0f7331157f0954be141be06cfa4f)